test/helpers/slack-mock.js

```diff
@@ -1,8 +1,22 @@
 class Slack_Mock {
   constructor(token, autoReconnect, autoMark) {
     this.token = token;
+
+    // Methods
     this.login = sandbox.spy();
     this.on = sandbox.spy();
+
+    // Data
+    this.self = {
```

Comment:
This nomenclature is a little confusing since `self` is often used to store a reference to a scope, is there another name that would suit this better?